x-pack/plugins/ml/public/maps/anomaly_source.tsx

```diff
@@ -147,8 +149,23 @@ export class AnomalySource implements IVectorSource {
     return null;
   }
 
-  getSourceStatus() {
-    return { tooltipContent: null, areResultsTrimmed: true };
+  getSourceStatus(sourceDataRequest?: DataRequest): SourceStatus {
+    const meta = sourceDataRequest ? sourceDataRequest.getMeta() : null;
+
+    if (meta?.areResultsTrimmed) {
+      return {
+        tooltipContent: i18n.translate('xpack.ml.maps.resultsTrimmedMsg', {
```

Comment:
Not related to this code, but is it possible to only add the Influencers section of the point tooltip if they are not already shown as a partition field. For example here the only influencer I have configured is `vehicle.id` so the empty Influencers section is redundant:

![image](https://user-images.githubusercontent.com/7405507/160603711-ca29c7f8-589d-40f2-a368-de5eb99a2633.png)